.github/workflows/update_draft_features_weekly.yml

```diff
@@ -0,0 +1,32 @@
+name: Update Draft Features
+
+on:
+  schedule:
+# Runs at midnight on Mondays 05:30 UTC   
```

Comment:
Yep, that's as good a time as any! I'll also add `workflow_dispatch` so that we can trigger this manually at any time. Often useful for ensuring everything works after a change to the script or workflow.